lib/main.js

```diff
@@ -42,9 +42,22 @@ export default {
         const file = activeEditor.getPath();
         const cwd = path.dirname(file);
         const args = ["-wc", file];
-        return helpers.exec(command, args, {stream: "stderr", cwd: cwd}).then(output =>
-          helpers.parse(output, regex)
-        );
+        return helpers.exec(command, args, {stream: "stderr", cwd: cwd}).then(output => {
+          const toReturn = [];
+          output.split(/\r?\n/).forEach(function (line) {
+            const matches = regex.exec(line);
+            if (matches !== null) {
+              const msg = {
+                range: helpers.rangeFromLineNumber(activeEditor, Number.parseInt(matches[1])),
+                type: "Error",
+                text: matches[2],
+                filePath: file
+              };
+              toReturn.push(msg);
```

Comment:
Ah, an earlier version was attempting to use the extra text as a trace, so I needed to optionally add that if it existed. Since this is ignoring the trace code (for now), that would be simpler. I'll push up a change for that.